### PR TITLE
Update and improve mandarin translation

### PR DESCRIPTION
**Pull request by MASangys**
per title

### CODE REVIEW

**Review by biodranik — APPROVED** [review]
Thanks! Please sign the commit and it's good to go.

**Comment by MASangys** [comment]
> Thanks! Please sign the commit and it's good to go.

Sorry, but I cnnot sign it.

File: data/strings/strings.txt [hunk 1]
```diff
@@ -15779,7 +15789,7 @@
     tr = Bir açıklama yazın (metin veya html)
     uk = Додайте опис (текст чи html)
     vi = Đánh vào một mô tả (văn bản hoặc html)
-    zh-Hans = 请添加说明(文字或html)
+    zh-Hans = 请添加说明（文字或html）
```
Comment (rtsisyk) [inline data/strings/strings.txt:15792]: Trailing space here.

**Review by rtsisyk — APPROVED** [review]
Please add Signed-off-by.

git commit amend -s

File: data/strings/strings.txt (hunk 1 above)
Comment (MASangys) [reply data/strings/strings.txt:15792]: So just need to change the text by adding @@ -15779,7 +15789,7 @@ in first line?

**Pull request merged**